js/product-details.js

```diff
@@ -0,0 +1,155 @@
+var xmlhttp = new XMLHttpRequest();
+xmlhttp.onreadystatechange = function () {
+  if (this.readyState == 4 && this.status == 200) {
+    var data = JSON.parse(this.responseText);
+    var drink = data[productCategory][productId].additionalDescription;
+    for (key in drink) {
+      if (drink[key] === "") {
```

Comment:
no empty brackets
change to 
if (!drink[key]) 
if drink key could be 0 - add also || drink[key] === 0